test/components/multiplayer/Chat.spec.js

```diff
@@ -0,0 +1,334 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { defaults, noop } from 'lodash';
+
+import Chat from '../../../src/common/components/multiplayer/Chat';
+
+const getWrapper = (props) => {
+  const defaultProps = {
+    open: true,
+    fullscreen: false,
+    toggleChat: noop,
+    onSendMessage: noop,
+    inGame: false,
+    roomName: 'dummy name',
+    messages: []
+  };
+
+  defaults(props, defaultProps);
+  return shallow(<Chat
+    open={props.open}
+    fullscreen={props.fullscreen}
+    toggleChat={props.toggleChat}
+    onSendMessage={props.onSendMessage}
+    inGame={props.inGame}
+    messages={props.messages}
+    roomName={props.roomName}
+  />);
+};
+
+const makeMessage = (msg) => {
```

Comment:
(minor) Could you do `const makeMessage = (msg = {}) => {` to avoid having to `msg = msg || {}`?